micro-ui/web/micro-ui-internals/packages/modules/dss/CHANGELOG.md

```
@@ -1,4 +1,7 @@
 
+## [1.8.11]  [3-Feb-2025]
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)